source/api/commands/then.md

```diff
@@ -70,6 +72,17 @@ cy.get('button').then(($btn) => {
 
 ## Change subject
 
+### The subject is changed with another command
+
+```javascript
+cy.get('button').then(($btn) => {
+  const cls = $btn.class()
+
+  cy.wrap($btn).click().should('not.have.class', cls)
+    .find('i')
+}).should('have.class', 'spin') // assert on i element
+```
+
```

Comment:
I would also include "simple" `cy.wrap(1).then(() => cy.wrap(2)).should('equal', 2)` examples for each (return, no return, return Cypress command), I think they make it pretty clear what is going on